comics/aggregator/crawler.py

```diff
@@ -250,3 +250,25 @@ def crawl_helper(self, short_name, pub_date):
         page = self.parse_page(page_url)
         url = page.src('img[src*="/img/%s/%s"]' % (short_name, date_string))
         return CrawlerImage(url)
+
+
+class DagbladetCrawlerBase(CrawlerBase):
+    """Base comics crawler for all comics posted at dagbladet.no"""
+    headers = {'User-Agent': 'Mozilla/5.0'}
+    time_zone = 'Europe/Oslo'
+
+    def crawl_helper(self, short_name, pub_date):
+        epoch = self.date_to_epoch(pub_date)
+        page_url = 'http://www.dagbladet.no/tegneserie/%s' % (short_name)
```

Comment:
The parenthesis here have no effect. If you want to make it a tuple, add a trailing comma.